Find a functionally equivalent code:android.media.MediaRouter.selectRouteInt:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide internal use
 */

@@ -1,3 +1,3 @@
 {
-    selectRouteStatic(types, route);
+    selectRouteStatic(types, route, explicit);
 }

